Detectors/ITSMFT/ITS/simulation/src/V3Services.cxx

```diff
@@ -763,7 +763,10 @@ void V3Services::ibEndWheelSideC(const Int_t iLay, TGeoVolume* endWheel, const T
   endWheel->AddNode(endWheelCVol, 1, new TGeoCombiTrans(0, 0, -zpos, new TGeoRotation("", 0, 180, 0)));
 
   // The position of the Steps is given wrt the holes (see eg. ALIITSUP0187)
-  dphi = sEndWCStepHolePhi0[iLay];
+  if (iLay == 0)
+    dphi = (sEndWCStepYlow / sEndWCStepYdispl[iLay]) * TMath::RadToDeg();
+  else
+    dphi = sEndWCStepHolePhi0[iLay - 1];
```

Comment:
Hi @mario6829, please add {}, they will be anyway asked by the CI. Also, clang-format needs to be fixed.

Reply:
Done, thanks for spotting!